GuideForNewDevelopers.md

```diff
@@ -0,0 +1,559 @@
+
+```sh
+python --version
+```
+
+If you dont have a latest version of python, Please download and install latest python from the [Official Python Website](https://www.python.org/downloads/). 
+
+1. Navigate to the Downloads tab for Windows.
+2. Choose the latest Python 3 release
+3. Choose the Windows x86 executable installer if you are using a 32-bit installer or if you have a 64-bit system, then download Windows x86-64 executable installer. 
+4. Run the executable and install python with the default options
+   1. Dont forget to select the `Add Python x.x to PATH` option
+   
+   Here is a detailed [guide](https://realpython.com/installing-python/#how-to-install-python-on-windows) if you need further help
+
+Open cmd and enter the following commands to verify you have installed python3
+
+
+```sh
+python --version
+```
+
+### 1.2.2 Install git Bash
+Download git bash from the official website and install it with the default settings
+
+https://git-scm.com/download/win
+
+Git is free and open source software for distributed version control. We will use it later to clone the Physio-net project to our system, and you can also use it to submit your contribution to the project. 
+Here are few resources to learn about git [git-scm.com](https://git-scm.com/book/en/v2/Getting-Started-What-is-Git%3F), [w3schools](https://www.w3schools.com/git/)
+
+
+
+### 1.2.3 Install VSCode
+
+To setup, VSCode, you can download the .exe package from [official link](https://code.visualstudio.com/) and install directly on your Windows. [Here](https://code.visualstudio.com/docs/setup/windows) are the official instructions 
+
+
+After installing VSCode, you can install this [Python extension on VSCode](https://marketplace.visualstudio.com/items?itemName=ms-python.python) to get the support for IntelliSense (Pylance), Linting, Debugging (multi-threaded, remote), Jupyter Notebooks, code formatting, refactoring, unit tests, and more. 
+Simply vist the link above and click on the install button on the website, it should redirect and open the installation option on VSCode.
+
+### 1.2.4 Clone the Project Locally
+
+Now that we have Python setup, Lets go ahead and clone the project to your system. Open git bash and enter the following command.
+
+
+```sh
+git clone https://github.com/MIT-LCP/physionet-build
+```
+
+### 1.2.5 Install and Create Virtual Environment
+
+Now that we have cloned the project to your system, lets go ahead and create a Virtual Environment, you can learn more about Virtual Environments in python [Here](https://realpython.com/python-virtual-environments-a-primer/). 
+
+In Summary, Virtual Environment will let us install and keep different versions of python library specific to individual projects.
+
+Lets create a virtual environment for our project
+
+1.2.5.1 Create a Virtual Environment
+
+Open CMD and  navigate inside the project directory
+
+```sh
```

Comment:
the markup `sh` is shorthand for `shell`, which is different than `cmd` (the windows flavour if you like), i.e. the syntax will be highlighted differently.